client/component-lab/Notification/helpers.coffee

```diff
@@ -0,0 +1,35 @@
+Timer = (callback, delay) ->
+
+  timerId = undefined
+  start = undefined
+  remaining = delay
+
+  @pause = ->
+
+    clearTimeout timerId
+    remaining -= new Date - start
+
+
+  @resume = ->
+
+    start = new Date
+    clearTimeout timerId
+    timerId = setTimeout(callback, remaining)
+
+
+  @clear = ->
+
+    clearTimeout timerId
+
+
+  @resume()
+
+propsValidation = (notification, types) ->
+  if not notification.type
+    throw new Error('Notification type is required.')
+  if _.keys(types).indexOf(notification.type) is -1
```

Comment:
`lodash` again, this is the second time I notice this. This should break your own environment, do you push untested code?